### DISCUSSION TITLE
I clone babel then run yarn install, and an error appeals

### DISCUSSION THREAD

**Discussion by arichyx** [babel/babel#12039] [Q&A] [answered]
I want to explore the source code of babel, so I clone babel repo to my Mac then run `yarn install` at the root dir, but it throws an error about yarn.
What should I do?

![image](https://user-images.githubusercontent.com/29599723/92345338-9544e800-f0fb-11ea-899d-fd1951224eaf.png)

**Comment by JLHwung** [comment, marked as answer]
Babel uses Yarn 2 and to build babel you will need `node >= 10.16.0 || node >= 11.7.0`. Please upgrade your node versions.

**Reply by arichyx** [reply]
It works. Thanks!
